tests/test_applets/featureSelection/testNewFeatureSelection.py

```diff
@@ -0,0 +1,381 @@
+import logging
+import numpy
+import time
+import vigra
+
+from lazyflow.graph import Graph, OperatorWrapper
+
+from ilastik.applets.featureSelection.opFeatureSelection import OpFeatureSelection
+from lazyflow.operators.oldVigraOperators import OpFeatureSelection as OpFeatureSelectionOld
```

Comment:
should have asked it in the `lazyflow` PR, but why do you keep the `oldVigraOperators`?

Reply:
only for a short transition period, as it makes testing the new code much easier...